tools/ci_connector_ops/ci_connector_ops/pipelines/tests/__init__.py

```diff
@@ -87,13 +100,18 @@ async def run_connector_test_pipeline(context: ConnectorContext, semaphore: anyi
     """
     async with semaphore:
         async with context:
+            version_checks_results = await run_version_checks(context)
```

Comment:
Just thinking about the skip on failure login in `run_tests`, could we use that here?

Reply:
I agree but in in this context I think `run_steps` is not immediately compatible as I wrapped steps executions inside function like `run_versions_checks`, `run_qa_checks` etc. 
I think we might eventually want to make these function Steps themselves (or `Pipeline`). And a `Pipeline` could nest multiple other pipelines execution.